packages/react/src/components/Table/mdx/Table.mdx

```diff
@@ -533,6 +538,7 @@ import TableViewDropdownContent from '../TableViewDropdown/TableViewDropdownCont
 | hasOnlyPageData             | bool                                                     |          | if true, the data prop will be assumed to only represent the currently visible page                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | hasSearch                   | bool                                                     |          | if true, simple search capability is enabled                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | hasFastSearch               | bool                                                     | true     | If false, the onApplySearch callback will only be triggered when hitting 'Enter' or if the 'Clear search input' 'x' is clicked.                                                                                                                                                                                                                                                                                                                                                                                   |
+| pinColumn                   | 'none', 'first', 'last' .                                | 'none'   | Freezes selected column regardless of horizontal scrollbar position                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Does that imply that I need to "select" the column in order for it to be pinned?   Or should this message be...
Can freeze the 'first' or 'last' column regardless of horizontal scroll position.  Default is 'none'.

And does this work in RTL mode as well?

Reply:
Thanks for the note! I updated the doc comment.
Yes, it works in RTL mode as well.